server/test/routes/api/v1/users.test.js

```diff
@@ -0,0 +1,30 @@
+import { describe, it } from 'node:test';
+import * as assert from 'node:assert';
+import { build, nodemailerMock } from '../../../helper.js';
+
+describe('/api/v1/users', () => {
+  describe('POST /register', () => {
```

Comment:
Created an initial test for the new API endpoint. We should always create a comprehensive integration test suite for every new API endpoint.